Library/Formula/fb-infer.rb

```diff
@@ -0,0 +1,56 @@
+class FbInfer < Formula
+  desc "Static analyzer for Java, C and Objective-C"
+  homepage "http://fbinfer.com/"
+
+  url "https://github.com/facebook/infer/releases/download/v0.5.0/infer-osx-v0.5.0.tar.xz"
+  sha256 "6a8547ac0b75a5e2bbeccae2169e39f753a60adbcacb6c94599fd31343a71ce7"
+  revision 1
+  head "https://github.com/facebook/infer.git"
+
+  option "with-clang-only", "Installs Infer with only Clang support."
+  option "with-java-only", "Installs Infer with only Java support."
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "opam" => :build
+
+  def install
+    ENV.deparallelize
```

Comment:
This actually isn't needed, somehow just got added when I was debugging the script. I'll remove it.